scripts/link-workbench.sh

```
@@ -0,0 +1,123 @@
+# Generate unique version with timestamp
+TIMESTAMP=$(date +%s)
+DEV_VERSION="0.0.86-dev.$TIMESTAMP"
+
+# Function to update and restore package version
+update_version() {
+	local pkg_dir="$1"
+	local new_version="$2"
+	local pkg_file="$pkg_dir/package.json"
+	
+	# Store original version
+	local original_version=$(node -p "require('$pkg_file').version")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)